proto/tripcommon.proto

```diff
@@ -133,6 +133,8 @@ message StreetName {
 message TurnLane {
   optional uint32 directions_mask = 1;
   optional bool is_active = 2;
+  optional bool is_possible = 3;
```

Comment:
What do you think about refactoring `is_active` and `is_possible` into an enum? That way it's clear that there's a hierarchy of states Active->Possible->Invalid.

Reply:
I'm not sure how an enum conveys a hierarchy. It would club them together under a single type to convey they are related. If you feel that makes it more readable, we can change it to an enum. What do you think?